os/arch/arm/src/stm32l4/Make.defs

```diff
@@ -207,6 +212,15 @@ ifeq ($(CONFIG_STM32L4_HAVE_HSI48),y)
 CHIP_CSRCS += stm32l4_hsi48.c
 endif
 
+ifeq ($(CONFIG_PM),y)
+CHIP_CSRCS += stm32l4_pmlpr.c stm32l4_pmsleep.c stm32l4_pmstandby.c
```

Comment:
Is pmlpr required ?

Reply:
It is lpr mode for stm32l4. Currently we are not using it. I removed for now.